amanuensis/blueprints/project.py

```diff
@@ -43,35 +66,32 @@ def get_projetcs():
         tmp_project["id"] = project["id"]
         tmp_project["name"] = project["name"]
 
-        status_code = None
-        status = None
+        requests_status_codes = []
+        request_status_code = None
+        project_status_code = None
+        project_status_name = None
         status_date = None
         submitted_at = None
         completed_at = None
+
         for request in project["requests"]:
-            # get status
-            # "status": "", // string ("IN REVIEW" | "APPROVED" | "REJECTED")
-            # "submitted_at": "", // string (timestamp) or null
-            # "completed_at": "" // string (timestamp) or null
+            request_state = current_session.query(RequestState).filter(RequestState.request_id == request["id"]).all()[-1]
```

Comment:
-1 is getting the latest in the response? If so we may want to use a .order() to order the response by date.